Hibernate with postgres throws error

With new hibernate version we're getting this error on startup, which doesn't prevent server to start:
```
Caused by: java.sql.SQLFeatureNotSupportedException: Method org.postgresql.jdbc.PgConnection.createClob() is not yet implemented.
	at org.postgresql.Driver.notImplemented(Driver.java:688)
	at org.postgresql.jdbc.PgConnection.createClob(PgConnection.java:1269)
```
There's some info here how to silence it https://github.com/pgjdbc/pgjdbc/issues/1102

Looks like we're just going down a route to change default logging for `org.hibernate.engine.jdbc.env.internal.LobCreatorBuilderImpl` to WARN as that exception is logged on INFO level and is just notification and doesn't cause app to fail to start or function properly.

--- a/spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml
+++ b/spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml
@@ -17,6 +17,8 @@ logging:
     org.springframework.boot.autoconfigure.security: 'INFO'
     # The following INFO is to log the embedded tomcat port
     org.springframework.boot.web.embedded.tomcat: 'INFO'
+    # Silence annoying exception trace logged on info level - gh-2750
+    org.hibernate.engine.jdbc.env.internal.LobCreatorBuilderImpl: 'WARN'
 spring:
   data:
     redis:

